### ISSUE TITLE
Dynamic binding

### ISSUE THREAD

**Issue by khuongduybui** [MarcScheib/aurelia-syntax-highlighter#1]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
```

**Current behavior**

I have a simple setup like this:
```HTML
<textarea value.bind="test"></textarea>
<pre>${test}</pre>
<pre class="language-javascript" au-syntax>${test}</pre>
<pre class="language-javascript" au-syntax textcontent.bind="test"></pre>
```
```Javascript
class MyClass {
  test = '{"a": true}';
}
```
At first when the page loads, I observe that:
- The first ```pre``` shows the correct content with no formatting (OK)
- The second ```pre``` shows the correct content with proper formatting (OK)
- The third ```pre``` shows the correct content with proper formatting (OK)

When I change the content of the ```textarea```, I observe that:
- The first ```pre``` updates its content as expected, no formatting is applied (OK)
- The second ```pre``` DOES NOT update its content (**NOT OK**) but preserves formatting (OK)
- The third ```pre``` updates its content as expected (OK) but loses formatting (**NOT OK**)

**Expected behavior**
When I change the content of the ```textarea```, I observe that:
- The second ```pre``` updates its content as expected and preserves formatting
- The third ```pre``` updates its content as expected and preserves formatting

**Comment by MarcScheib** [comment]
I released a new version 0.2.0 which should solve your problem. You can do the following now:

```html
<pre><code class="language-javascript" au-syntax.bind="test"></code></pre>
```

**Comment by khuongduybui** [comment]
Thanks for the quick turnaround.
